build.gradle

```diff
@@ -64,6 +64,7 @@ gitProperties {
   gitPropertiesResourceDir = file("${rootDir}/build")
   extProperty = 'gitProps'
   failOnNoGitDirectory = true
+  keys = ['git.commit.id.abbrev', 'git.commit.id', 'git.branch', 'git.tags', 'git.build.version']
```

Comment:
I would probably also include `git.commit.message.short / git.commit.time` as that seems useful as well

Reply:
sure added, i just wanted to omit exposing the two props (git.build.host & git.remote.origin.url) hence added the change to only allow props required by us.

Reply:
+1 on omitting those two.